pipelines/PingCAP-QE/tidb-test/latest/ghpr_integration_nodejs_test.groovy

```diff
@@ -78,25 +78,25 @@ pipeline {
                 }
             }
         }
-        stage('TypeORM Tests') {
+        stage('Node.js Tests') {
             matrix {
                 axes {
                     axis {
                         name 'TEST_PARAMS'
-                        values 'typeorm_test ./test.sh'
+                        values 'prisma_test ./test.sh', 'typeorm_test ./test.sh', 'sequelize_test ./test.sh'
```

Comment:
Suggest making a modification, directly adding the execution cmds `cd prisma_test && chmod +x *.sh && ./test.sh`

Reply:
Great. More simple